nixos/modules/misc/ids.nix

```diff
@@ -638,6 +638,7 @@
       rss2email = 312;
       cockroachdb = 313;
       zoneminder = 314;
+      clickshare = 315;
```

Comment:
No need to use a static gid, just define the group like `users.groups.clickshare = {}` and it will automatically be assigned a gid

Reply:
Cool, thanks, I didn't know that.
Branch got updated.